[CI] DocsClientYamlTestSuiteIT {yaml=reference/search/async-search/submit-async-search-date-histogram-example}  failing

`DocsClientYamlTestSuiteIT.test {yaml=reference/search/async-search/submit-async-search-date-histogram-example}` has been regularly failing since it was added recently.

```
org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT > test {yaml=reference/search/async-search/submit-async-search-date-histogram-example} FAILED
    java.lang.AssertionError: Failure at [reference/search/async-search:63]: $body didn't match expected value:
                             $body: 
           expiration_time_in_millis: same [1585153485758]
                                  id: same [FjlsX1pQMm1qU0ZxYVppU3VIQjlLTHccRVJFbjBIZXVUNHFCMm...]
                          is_partial: same [true]
                          is_running: same [true]
                            response: 
                               _shards: 
                                  failed: same [0]
                                 skipped: same [0]
                              successful: same [2]
                                   total: same [2]
                                  hits: 
                                    hits: same [empty list]
                               max_score: same [null]
                                   total: 
                                  relation: expected String [gte] but was String [eq]
                                     value: same [7]
                     num_reduce_phases: expected [0] but not found
                             timed_out: same [false]
                                  took: same [20]
                          aggregations: unexpected but found [{sale_date={buckets=[{key_as_string=2015/01/01 00:00:00, key=1420070400000, doc_count=3}, {key_as_string=2015/01/02 00:00:00, key=1420156800000, doc_count=0}, {key_as_string=2015/01/03 00:00:00, key=1420243200000, doc_count=0}, {key_as_string=2015/01/04 00:00:00, key=1420329600000, doc_count=0}, {key_as_string=2015/01/05 00:00:00, key=1420416000000, doc_count=0}, {key_as_string=2015/01/06 00:00:00, key=1420502400000, doc_count=0}, {key_as_string=2015/01/07 00:00:00, key=1420588800000, doc_count=0}, {key_as_string=2015/01/08 00:00:00, key=1420675200000, doc_count=0}, {key_as_string=2015/01/09 00:00:00, key=1420761600000, doc_count=0}, {key_as_string=2015/01/10 00:00:00, key=1420848000000, doc_count=0}, {key_as_string=2015/01/11 00:00:00, key=1420934400000, doc_count=0}, {key_as_string=2015/01/12 00:00:00, key=1421020800000, doc_count=0}, {key_as_string=2015/01/13 00:00:00, key=1421107200000, doc_count=0}, {key_as_string=2015/01/14 00:00:00, key=1421193600000, doc_count=0}, {key_as_string=2015/01/15 00:00:00, key=1421280000000, doc_count=0}, {key_as_string=2015/01/16 00:00:00, key=1421366400000, doc_count=0}, {key_as_string=2015/01/17 00:00:00, key=1421452800000, doc_count=0}, {key_as_string=2015/01/18 00:00:00, key=1421539200000, doc_count=0}, {key_as_string=2015/01/19 00:00:00, key=1421625600000, doc_count=0}, {key_as_string=2015/01/20 00:00:00, key=1421712000000, doc_count=0}, {key_as_string=2015/01/21 00:00:00, key=1421798400000, doc_count=0}, {key_as_string=2015/01/22 00:00:00, key=1421884800000, doc_count=0}, {key_as_string=2015/01/23 00:00:00, key=1421971200000, doc_count=0}, {key_as_string=2015/01/24 00:00:00, key=1422057600000, doc_count=0}, {key_as_string=2015/01/25 00:00:00, key=1422144000000, doc_count=0}, {key_as_string=2015/01/26 00:00:00, key=1422230400000, doc_count=0}, {key_as_string=2015/01/27 00:00:00, key=1422316800000, doc_count=0}, {key_as_string=2015/01/28 00:00:00, key=1422403200000, doc_count=0}, {key_as_string=2015/01/29 00:00:00, key=1422489600000, doc_count=0}, {key_as_string=2015/01/30 00:00:00, key=1422576000000, doc_count=0}, {key_as_string=2015/01/31 00:00:00, key=1422662400000, doc_count=0}, {key_as_string=2015/02/01 00:00:00, key=1422748800000, doc_count=2}, {key_as_string=2015/02/02 00:00:00, key=1422835200000, doc_count=0}, {key_as_string=2015/02/03 00:00:00, key=1422921600000, doc_count=0}, {key_as_string=2015/02/04 00:00:00, key=1423008000000, doc_count=0}, {key_as_string=2015/02/05 00:00:00, key=1423094400000, doc_count=0}, {key_as_string=2015/02/06 00:00:00, key=1423180800000, doc_count=0}, {key_as_string=2015/02/07 00:00:00, key=1423267200000, doc_count=0}, {key_as_string=2015/02/08 00:00:00, key=1423353600000, doc_count=0}, {key_as_string=2015/02/09 00:00:00, key=1423440000000, doc_count=0}, {key_as_string=2015/02/10 00:00:00, key=1423526400000, doc_count=0}, {key_as_string=2015/02/11 00:00:00, key=1423612800000, doc_count=0}, {key_as_string=2015/02/12 00:00:00, key=1423699200000, doc_count=0}, {key_as_string=2015/02/13 00:00:00, key=1423785600000, doc_count=0}, {key_as_string=2015/02/14 00:00:00, key=1423872000000, doc_count=0}, {key_as_string=2015/02/15 00:00:00, key=1423958400000, doc_count=0}, {key_as_string=2015/02/16 00:00:00, key=1424044800000, doc_count=0}, {key_as_string=2015/02/17 00:00:00, key=1424131200000, doc_count=0}, {key_as_string=2015/02/18 00:00:00, key=1424217600000, doc_count=0}, {key_as_string=2015/02/19 00:00:00, key=1424304000000, doc_count=0}, {key_as_string=2015/02/20 00:00:00, key=1424390400000, doc_count=0}, {key_as_string=2015/02/21 00:00:00, key=1424476800000, doc_count=0}, {key_as_string=2015/02/22 00:00:00, key=1424563200000, doc_count=0}, {key_as_string=2015/02/23 00:00:00, key=1424649600000, doc_count=0}, {key_as_string=2015/02/24 00:00:00, key=1424736000000, doc_count=0}, {key_as_string=2015/02/25 00:00:00, key=1424822400000, doc_count=0}, {key_as_string=2015/02/26 00:00:00, key=1424908800000, doc_count=0}, {key_as_string=2015/02/27 00:00:00, key=1424995200000, doc_count=0}, {key_as_string=2015/02/28 00:00:00, key=1425081600000, doc_count=0}, {key_as_string=2015/03/01 00:00:00, key=1425168000000, doc_count=2}]}}]
                start_time_in_millis: same [1584721485758]
                             version: expected Integer [0] but was Integer [1]
```

https://gradle-enterprise.elastic.co/s/qobww2xln2vp6/tests/jbm2x6tnwtxku-5b4duzugn4e5w

Looks to be limited to `master` and did not reproduce for me locally.

--- a/docs/reference/search/async-search.asciidoc
+++ b/docs/reference/search/async-search.asciidoc
@@ -178,7 +178,7 @@ GET /_async_search/FmRldE8zREVEUzA2ZVpUeGs2ejJFUFEaMkZ5QTVrSTZSaVN3WlNFVmtlWHJsd
 // TESTRESPONSE[s/"successful" : 188/"successful": $body.response._shards.successful/]
 // TESTRESPONSE[s/"value" : 456433/"value": $body.response.hits.total.value/]
 // TESTRESPONSE[s/"buckets" : \[\]/"buckets": $body.response.aggregations.sale_date.buckets/]
-// TESTRESPONSE[s/"num_reduce_phases" : 38,//]
+// TESTRESPONSE[s/"num_reduce_phases" : 46,//]
 
 <1> Whether the returned search results are partial or final
 <2> Whether the search is still being executed or it has completed

--- a/docs/reference/search/async-search.asciidoc
+++ b/docs/reference/search/async-search.asciidoc
@@ -68,7 +68,6 @@ results are returned as part of the <<search-api-response-body,`response`>> obje
 }
 --------------------------------------------------
 // TESTRESPONSE[s/FmRldE8zREVEUzA2ZVpUeGs2ejJFUFEaMkZ5QTVrSTZSaVN3WlNFVmtlWHJsdzoxMDc=/$body.id/]
-// TESTRESPONSE[s/"version" : 0/"version": $body.version/]
 // TESTRESPONSE[s/"is_partial" : true/"is_partial": $body.is_partial/]
 // TESTRESPONSE[s/"is_running" : true/"is_running": $body.is_running/]
 // TESTRESPONSE[s/1583945890986/$body.start_time_in_millis/]
@@ -170,7 +169,6 @@ GET /_async_search/FmRldE8zREVEUzA2ZVpUeGs2ejJFUFEaMkZ5QTVrSTZSaVN3WlNFVmtlWHJsd
 }
 --------------------------------------------------
 // TESTRESPONSE[s/FmRldE8zREVEUzA2ZVpUeGs2ejJFUFEaMkZ5QTVrSTZSaVN3WlNFVmtlWHJsdzoxMDc=/$body.id/]
-// TESTRESPONSE[s/"version" : 39/"version" : $body.version/]
 // TESTRESPONSE[s/"is_partial" : true/"is_partial" : false/]
 // TESTRESPONSE[s/"is_running" : true/"is_running" : false/]
 // TESTRESPONSE[s/1583945890986/$body.start_time_in_millis/]

--- a/docs/reference/search/async-search.asciidoc
+++ b/docs/reference/search/async-search.asciidoc
@@ -146,7 +146,7 @@ GET /_async_search/FmRldE8zREVEUzA2ZVpUeGs2ejJFUFEaMkZ5QTVrSTZSaVN3WlNFVmtlWHJsd
   "response" : {
     "took" : 12144,
     "timed_out" : false,
-    "num_reduce_phases" : 38, <4>
+    "num_reduce_phases" : 46, <4>
     "_shards" : {
       "total" : 562, <5>
       "successful" : 188,

--- a/docs/reference/search/async-search.asciidoc
+++ b/docs/reference/search/async-search.asciidoc
@@ -30,9 +30,8 @@ POST /sales*/_async_search?size=0
     }
 }
 --------------------------------------------------
-// TEST[skip:"AwaitsFix https://github.com/elastic/elasticsearch/issues/53887"]
 // TEST[setup:sales]
-// TEST[s/size=0/size=0&wait_for_completion=0/]
+// TEST[s/size=0/size=0&wait_for_completion=10s&clean_on_completion=false/]
 
 The response contains an identifier of the search being executed.
 You can use this ID to later retrieve the search's final results.
@@ -70,12 +69,18 @@ results are returned as part of the <<search-api-response-body,`response`>> obje
 }
 --------------------------------------------------
 // TESTRESPONSE[s/FmRldE8zREVEUzA2ZVpUeGs2ejJFUFEaMkZ5QTVrSTZSaVN3WlNFVmtlWHJsdzoxMDc=/$body.id/]
+// TESTRESPONSE[s/"version" : 0/"version": $body.version/]
+// TESTRESPONSE[s/"is_partial" : true/"is_partial": $body.is_partial/]
+// TESTRESPONSE[s/"is_running" : true/"is_running": $body.is_running/]
 // TESTRESPONSE[s/1583945890986/$body.start_time_in_millis/]
 // TESTRESPONSE[s/1584377890986/$body.expiration_time_in_millis/]
 // TESTRESPONSE[s/"took" : 1122/"took": $body.response.took/]
+// TESTRESPONSE[s/"num_reduce_phases" : 0,//]
 // TESTRESPONSE[s/"total" : 562/"total": $body.response._shards.total/]
 // TESTRESPONSE[s/"successful" : 3/"successful": $body.response._shards.successful/]
 // TESTRESPONSE[s/"value" : 157483/"value": $body.response.hits.total.value/]
+// TESTRESPONSE[s/"relation" : "gte"/"relation": $body.response.hits.total.relation/]
+// TESTRESPONSE[s/"hits" : \[ \]\n\s\s\s\s\}/"hits" : \[\]},"aggregations":  $body.response.aggregations/]
 
 <1> Identifier of the async search that can be used to monitor its progress, retrieve its results, and/or delete it.
 <2> Whether the returned search results are partial or final
@@ -135,7 +140,7 @@ GET /_async_search/FmRldE8zREVEUzA2ZVpUeGs2ejJFUFEaMkZ5QTVrSTZSaVN3WlNFVmtlWHJsd
 --------------------------------------------------
 {
   "id" : "FmRldE8zREVEUzA2ZVpUeGs2ejJFUFEaMkZ5QTVrSTZSaVN3WlNFVmtlWHJsdzoxMDc=",
-  "version" : 2, <1>
+  "version" : 39, <1>
   "is_partial" : true, <2>
   "is_running" : true, <3>
   "start_time_in_millis" : 1583945890986,

--- a/docs/reference/search/async-search.asciidoc
+++ b/docs/reference/search/async-search.asciidoc
@@ -163,7 +163,7 @@ GET /_async_search/FmRldE8zREVEUzA2ZVpUeGs2ejJFUFEaMkZ5QTVrSTZSaVN3WlNFVmtlWHJsd
       "max_score" : null,
       "hits" : [ ]
     },
-    "aggregations" : { <5>
+    "aggregations" : { <5>ok
       "sale_date" :  {
         "buckets" : []
       }
@@ -172,6 +172,7 @@ GET /_async_search/FmRldE8zREVEUzA2ZVpUeGs2ejJFUFEaMkZ5QTVrSTZSaVN3WlNFVmtlWHJsd
 }
 --------------------------------------------------
 // TESTRESPONSE[s/FmRldE8zREVEUzA2ZVpUeGs2ejJFUFEaMkZ5QTVrSTZSaVN3WlNFVmtlWHJsdzoxMDc=/$body.id/]
+// TESTRESPONSE[s/"version" : 39/"version" : $body.version/]
 // TESTRESPONSE[s/"is_partial" : true/"is_partial" : false/]
 // TESTRESPONSE[s/"is_running" : true/"is_running" : false/]
 // TESTRESPONSE[s/1583945890986/$body.start_time_in_millis/]

